.travis.yml

```diff
@@ -8,11 +8,10 @@ php:
   - nightly
 
 install:
-  - wget https://phar.phpunit.de/phpunit-old.phar
   - composer install
 
 script:
-  - php phpunit-old.phar
```

Comment:
I back to that issue due to phpunit 6 namespaces. Are you still planning to maintain phpunit as dependency but not included in your composer.json?